RPi/_GPIO.py

```diff
@@ -33,17 +31,62 @@
 BOARD   = 2
 
 # Output modes
-LOW  = 0
-HIGH = 1
+LOW  = gpiod.Line.ACTIVE_LOW
+HIGH = gpiod.Line.ACTIVE_HIGH
+
+_LINE_ACTIVE_STATE_COSNT_TO_FLAG = {
+    LOW: gpiod.LINE_REQ_FLAG_ACTIVE_LOW,
+    HIGH: 0,  # Active High is set by the default flag
+}
+
+
+# Macro
+def active_flag(const):
+    return _LINE_ACTIVE_STATE_COSNT_TO_FLAG[const]
+
+
+# We map RPi.GPIO PUD modes to libgpiod PUD constants
+PUD_OFF     = gpiod.Line.BIAS_AS_IS
```

Comment:
Hmmm, I've never looked at any libgpiod code, but shouldn't `PUD_OFF` map to `gpiod.Line.BIAS_DISABLE` ?